programs/drift/src/controller/orders.rs

```diff
@@ -978,7 +1120,7 @@ pub fn fill_perp_order(
     let oracle_twap_5min: i64;
     let perp_market_index: u16;
 
-    let mut amm_is_available = !state.amm_paused()?;
+    let mut amm_is_available = !state.amm_paused()? && fill_mode != FillMode::RFQ;
```

Comment:
nit: since we check this equality multiple times, would be nice to have an is_rfq() method on fill mode